Block/JsProductPage.php

```diff
@@ -165,4 +165,18 @@ public function isSaveHintsInSections()
     {
         return $this->featureSwitches->isSaveHintsInSections();
     }
+
+    public function isBoltProductPage()
+    {
+        if ($this->configHelper->getSelectProductPageCheckoutFlag())
+        {
+            $attributes = $this->getProduct()->getAttributes();
+            $codes = [];
+            foreach ($attributes as $attribute) {
+                array_push($codes, $attribute->getName());
```

Comment:
We don't need other attributes so maybe we can do early return once we find 'bolt_ppc'

Reply:
The `getData` approach we discussed offline didn't work, but I've implemented a quicker check now